### PR TITLE
Update 2025 dealers room

### PR DESCRIPTION
**Pull request by RandornCanis**

### CODE REVIEW

File: content/Dealers & Artists/drip/index.md [hunk 1]
```diff
@@ -11,328 +11,336 @@ banner_author: "Blue Hasia"
...
+Table or Memberships cancellations should be directed to both <dealers@anthrocon.org> and <refunds@anthrocon.org>. If canceling a table, please mention explicitly whether you wish to cancel your Dealer membership or have it converted to a standard Membership.
 
-Full refunds will be given for tables that are canceled on or before April 30th, 2024. **No refunds will be given for tables canceled after this.**
+Full refunds will be given for 2025 tables that are canceled on or before April 30th, 2025. No refunds will be given for tables canceled after this date!
 
-Full refunds will be given for 2023 memberships that are canceled on or before June 14th, 2024. **No refunds will be given for memberships canceled after this.** Dealerships can be ‘rolled over’ to the next year’s convention.
+Full refunds will be given for 2025 Memberships that are canceled on or before June 8th, 2025. No refunds will be given for memberships canceled after this date! Neither Memberships nor Dealerships can be ‘rolled over’ to the next year’s convention, and ‘vouchers’ are no longer provided.
```
Comment (RandornCanis) [inline content/Dealers & Artists/drip/index.md:258]: Confirm full refund or partial in line with general refund policy.

Comment (RandornCanis) [inline content/Dealers & Artists/drip/index.md:258]: Confirm three dates.

Comment (ronnienoodles) [reply content/Dealers & Artists/drip/index.md:258]: Pre-registration will close June 15th, 2025
Table cancellation refund date of April 30th is accurate
Membership refund date needs to be changed to June 13th, 2025

Comment (ronnienoodles) [reply content/Dealers & Artists/drip/index.md:258]: Full refund is applicable.

**Pull request merged**